RELEASE.md

```diff
@@ -10,6 +10,8 @@ Please follow the established format:
 
 ## Major features and improvements
 
+- Add support for Python 3.13
+- Add support for Python 3.12
```

Comment:
maybe after merge we put this in one line with both PR numbers seperated by commas